Uberjar building issue (0.26.2)

Dear all,

while trying to build the uberjar, an exception is thrown and the process is not concluded. My environment is as follows:

`
rodrigo@ubuntu:~/Projetos/metabase$ java -version
java version "1.8.0_144"
Java(TM) SE Runtime Environment (build 1.8.0_144-b01)
Java HotSpot(TM) 64-Bit Server VM (build 25.144-b01, mixed mode)

rodrigo@ubuntu:~/Projetos/metabase$ node -v
v8.5.0

rodrigo@ubuntu:~/Projetos/metabase$ yarn -v
1.2.0

rodrigo@ubuntu:~/Projetos/metabase$ lein -v
Leiningen 2.7.1 on Java 1.8.0_144 Java HotSpot(TM) 64-Bit Server VM
`

Running **lein ring server** compiles and runs flawlessy.

Tried with branches **0.26.1** and **0.26.2**.

The exception is:

`
rodrigo@ubuntu:~/Projetos/metabase$ bin/build
Tagging uberjar with version 'v0.26.2 9914bbb HEAD 2017-10-20'...
Running 'yarn' to download javascript dependencies...
yarn install v1.2.0
$ echo $npm_execpath | grep -q yarn || echo '\033[0;33mSorry, npm is not supported. Please use Yarn (https://yarnpkg.com/).\033[0m'
[1/5] Validating package.json...
[2/5] Resolving packages...
success Already up-to-date.
Done in 1.38s.
Running 'webpack' with NODE_ENV=production assemble and minify frontend assets...


Hash: 2d3a96e8f2afc460ba97
Version: webpack 1.15.0
Time: 326875ms
                                                     Asset       Size  Chunks             Chunk Names
 app-embed.bundle.css.map?9a5cbcef9490629326f125651b4a7092  130 bytes       0  [emitted]  app-embed
                  app-embed.bundle.js?2d3a96e8f2afc460ba97     3.3 MB       0  [emitted]  app-embed
                        2.2.bundle.js?2d3a96e8f2afc460ba97    13.8 kB       2  [emitted]  
                   app-main.bundle.js?2d3a96e8f2afc460ba97    4.92 MB       3  [emitted]  app-main
                        4.4.bundle.js?2d3a96e8f2afc460ba97    62.4 kB       4  [emitted]  
                 app-public.bundle.js?2d3a96e8f2afc460ba97    3.31 MB       5  [emitted]  app-public
                     styles.bundle.js?2d3a96e8f2afc460ba97    2.63 kB       6  [emitted]  styles
     app-embed.bundle.css?9a5cbcef9490629326f125651b4a7092    65.5 kB       0  [emitted]  app-embed
      app-main.bundle.css?4960c0b241f40d9cded7caadbef2f7d4    80.1 kB       3  [emitted]  app-main
    app-public.bundle.css?9a5cbcef9490629326f125651b4a7092    65.5 kB       5  [emitted]  app-public
        styles.bundle.css?d9d395a661f91d50dd647a49014a71dd    84.6 kB       6  [emitted]  styles
              app-embed.bundle.js.map?2d3a96e8f2afc460ba97    19.7 MB       0  [emitted]  app-embed
                        1.1.bundle.js?2d3a96e8f2afc460ba97    67.5 kB       1  [emitted]  
                    1.1.bundle.js.map?2d3a96e8f2afc460ba97     252 kB       1  [emitted]  
                    2.2.bundle.js.map?2d3a96e8f2afc460ba97    89.8 kB       2  [emitted]  
               app-main.bundle.js.map?2d3a96e8f2afc460ba97    27.5 MB       3  [emitted]  app-main
  app-main.bundle.css.map?4960c0b241f40d9cded7caadbef2f7d4  129 bytes       3  [emitted]  app-main
                    4.4.bundle.js.map?2d3a96e8f2afc460ba97    64.2 kB       4  [emitted]  
             app-public.bundle.js.map?2d3a96e8f2afc460ba97    19.7 MB       5  [emitted]  app-public
app-public.bundle.css.map?9a5cbcef9490629326f125651b4a7092  131 bytes       5  [emitted]  app-public
                 styles.bundle.js.map?2d3a96e8f2afc460ba97    12.6 kB       6  [emitted]  styles
    styles.bundle.css.map?d9d395a661f91d50dd647a49014a71dd  127 bytes       6  [emitted]  styles
                                          ../../index.html    4.17 kB          [emitted]  
                                         ../../public.html    4.17 kB          [emitted]  
                                          ../../embed.html    4.17 kB          [emitted]  
[1188] ./public/components ^\.\/.*$ 976 bytes {0} {3} {5} [built]
[2054] ./components ^(.*\.info\.(js$))[^.]*$/gi 255 bytes {4} [built]
    + 2069 hidden modules

WARNING in ./public/components/EmbedFrame.jsx
Critical dependencies:
88:16-23 require function is used in a way in which dependencies cannot be statically extracted
 @ ./public/components/EmbedFrame.jsx 88:16-23

WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback

WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/index.css
postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback

WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/index.css
postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/query_builder.css:90:5: variable '--purple-light-color' is undefined and used without a fallback

WARNING in 
UnusedFilesWebpackPlugin found some unused files:
components/AddButton.jsx
components/ConstrainToScreen.jsx
components/DeleteQuestionModal.jsx
components/Sidebar.jsx
internal/lib/components-node.js
public/lib/types.js
query_builder/components/CardFiltersWidget.js
xray/components/XrayFieldComparison.jsx

ERROR in 4.4.bundle.js?2d3a96e8f2afc460ba97 from UglifyJs
SyntaxError: Unexpected token: operator (>) [/home/rodrigo/Projetos/metabase/~/stringify-object/index.js:6,0]
Child html-webpack-plugin for "../../index.html":
        + 3 hidden modules
Child html-webpack-plugin for "../../public.html":
        + 3 hidden modules
Child html-webpack-plugin for "../../embed.html":
        + 3 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 6 hidden modules
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 8 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 9 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 8 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 7 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/core/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
Child extract-text-webpack-plugin:
        + 6 hidden modules
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/core/rounded.css:10:3: variable '--med-border-radius' is undefined and used without a fallback
    
    WARNING in /home/rodrigo/Projetos/metabase/~/css-loader?{"localIdentName":"[hash:base64:5]","restructuring":false,"compatibility":true,"url":false,"importLoaders":1}!/home/rodrigo/Projetos/metabase/~/postcss-loader!./css/index.css
    postcss-custom-properties: /home/rodrigo/Projetos/metabase/frontend/src/metabase/css/query_builder.css:90:5: variable '--purple-light-color' is undefined and used without a fallback
Sample Dataset already generated.
Running 'lein uberjar'...
Compiling metabase.driver.bigquery
log4j:ERROR Could not instantiate class [metabase.logger.Appender].
java.lang.ClassNotFoundException: metabase.logger.Appender
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:335)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at org.apache.log4j.helpers.Loader.loadClass(Loader.java:198)
	at org.apache.log4j.helpers.OptionConverter.instantiateByClassName(OptionConverter.java:327)
	at org.apache.log4j.helpers.OptionConverter.instantiateByKey(OptionConverter.java:124)
	at org.apache.log4j.PropertyConfigurator.parseAppender(PropertyConfigurator.java:785)
	at org.apache.log4j.PropertyConfigurator.parseCategory(PropertyConfigurator.java:768)
	at org.apache.log4j.PropertyConfigurator.configureRootCategory(PropertyConfigurator.java:648)
	at org.apache.log4j.PropertyConfigurator.doConfigure(PropertyConfigurator.java:514)
	at org.apache.log4j.PropertyConfigurator.doConfigure(PropertyConfigurator.java:580)
	at org.apache.log4j.helpers.OptionConverter.selectAndConfigure(OptionConverter.java:526)
	at org.apache.log4j.LogManager.<clinit>(LogManager.java:127)
	at org.slf4j.impl.Log4jLoggerFactory.<init>(Log4jLoggerFactory.java:66)
	at org.slf4j.impl.StaticLoggerBinder.<init>(StaticLoggerBinder.java:72)
	at org.slf4j.impl.StaticLoggerBinder.<clinit>(StaticLoggerBinder.java:45)
	at org.slf4j.LoggerFactory.bind(LoggerFactory.java:150)
	at org.slf4j.LoggerFactory.performInitialization(LoggerFactory.java:124)
	at org.slf4j.LoggerFactory.getILoggerFactory(LoggerFactory.java:412)
	at org.slf4j.LoggerFactory.getLogger(LoggerFactory.java:357)
	at clojure.tools.logging$eval219$reify__220.get_logger(logging.clj:288)
	at metabase.util$fn__3460.invokeStatic(util.clj:32)
	at metabase.util$fn__3460.invoke(util.clj:32)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$load_one.invoke(core.clj:5692)
	at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
	at clojure.core$load_lib.invokeStatic(core.clj:5736)
	at clojure.core$load_lib.doInvoke(core.clj:5717)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$load_libs.invokeStatic(core.clj:5778)
	at clojure.core$load_libs.doInvoke(core.clj:5758)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$require.invokeStatic(core.clj:5796)
	at clojure.core$require.doInvoke(core.clj:5796)
	at clojure.lang.RestFn.invoke(RestFn.java:551)
	at metabase.db$loading__5569__auto____2302.invoke(db.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$load_one.invoke(core.clj:5692)
	at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
	at clojure.core$load_lib.invokeStatic(core.clj:5736)
	at clojure.core$load_lib.doInvoke(core.clj:5717)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$load_libs.invokeStatic(core.clj:5778)
	at clojure.core$load_libs.doInvoke(core.clj:5758)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$require.invokeStatic(core.clj:5796)
	at clojure.core$require.doInvoke(core.clj:5796)
	at clojure.lang.RestFn.invoke(RestFn.java:512)
	at metabase.models.database$loading__5569__auto____1993.invoke(database.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$load_one.invoke(core.clj:5692)
	at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
	at clojure.core$load_lib.invokeStatic(core.clj:5736)
	at clojure.core$load_lib.doInvoke(core.clj:5717)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$load_libs.invokeStatic(core.clj:5778)
	at clojure.core$load_libs.doInvoke(core.clj:5758)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$require.invokeStatic(core.clj:5796)
	at clojure.core$require.doInvoke(core.clj:5796)
	at clojure.lang.RestFn.invoke(RestFn.java:703)
	at metabase.driver$loading__5569__auto____1124.invoke(driver.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$load_one.invoke(core.clj:5692)
	at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
	at clojure.core$load_lib.invokeStatic(core.clj:5736)
	at clojure.core$load_lib.doInvoke(core.clj:5717)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$load_libs.invokeStatic(core.clj:5778)
	at clojure.core$load_libs.doInvoke(core.clj:5758)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$require.invokeStatic(core.clj:5796)
	at clojure.core$require.doInvoke(core.clj:5796)
	at clojure.lang.RestFn.invoke(RestFn.java:930)
	at metabase.driver.bigquery$loading__5569__auto____36.invoke(bigquery.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$compile$fn__5682.invoke(core.clj:5903)
	at clojure.core$compile.invokeStatic(core.clj:5903)
	at clojure.core$compile.invoke(core.clj:5895)
	at user$eval20$fn__29.invoke(form-init82057072006848720.clj:1)
	at user$eval20.invokeStatic(form-init82057072006848720.clj:1)
	at user$eval20.invoke(form-init82057072006848720.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:6927)
	at clojure.lang.Compiler.eval(Compiler.java:6917)
	at clojure.lang.Compiler.eval(Compiler.java:6917)
	at clojure.lang.Compiler.load(Compiler.java:7379)
	at clojure.lang.Compiler.loadFile(Compiler.java:7317)
	at clojure.main$load_script.invokeStatic(main.clj:275)
	at clojure.main$init_opt.invokeStatic(main.clj:277)
	at clojure.main$init_opt.invoke(main.clj:277)
	at clojure.main$initialize.invokeStatic(main.clj:308)
	at clojure.main$null_opt.invokeStatic(main.clj:342)
	at clojure.main$null_opt.invoke(main.clj:339)
	at clojure.main$main.invokeStatic(main.clj:421)
	at clojure.main$main.doInvoke(main.clj:384)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at clojure.lang.Var.invoke(Var.java:383)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.Var.applyTo(Var.java:700)
	at clojure.main.main(main.java:37)
log4j:ERROR Could not instantiate appender named "metabase".
10-20 10:18:18 INFO metabase.util :: Loading Metabase...
10-20 10:18:27 INFO util.encryption :: DB details encryption is DISABLED for this Metabase instance. 🔓 
See http://www.metabase.com/docs/latest/operations-guide/start.html#encrypting-your-database-connection-details-at-rest for more information.
java.lang.ClassNotFoundException: metabase.query_processor.interface.Field, compiling:(generic_sql.clj:1:1)
Exception in thread "main" java.lang.ClassNotFoundException: metabase.query_processor.interface.Field, compiling:(generic_sql.clj:1:1)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3657)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$load_one.invoke(core.clj:5692)
	at clojure.core$load_lib$fn__5626.invoke(core.clj:5737)
	at clojure.core$load_lib.invokeStatic(core.clj:5736)
	at clojure.core$load_lib.doInvoke(core.clj:5717)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$load_libs.invokeStatic(core.clj:5778)
	at clojure.core$load_libs.doInvoke(core.clj:5758)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:648)
	at clojure.core$require.invokeStatic(core.clj:5796)
	at clojure.core$require.doInvoke(core.clj:5796)
	at clojure.lang.RestFn.invoke(RestFn.java:930)
	at metabase.driver.bigquery$loading__5569__auto____36.invoke(bigquery.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	at clojure.lang.Compiler.compile1(Compiler.java:7474)
	at clojure.lang.Compiler.compile1(Compiler.java:7464)
	at clojure.lang.Compiler.compile(Compiler.java:7541)
	at clojure.lang.RT.compile(RT.java:406)
	at clojure.lang.RT.load(RT.java:451)
	at clojure.lang.RT.load(RT.java:419)
	at clojure.core$load$fn__5677.invoke(core.clj:5893)
	at clojure.core$load.invokeStatic(core.clj:5892)
	at clojure.core$load.doInvoke(core.clj:5876)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5697)
	at clojure.core$compile$fn__5682.invoke(core.clj:5903)
	at clojure.core$compile.invokeStatic(core.clj:5903)
	at clojure.core$compile.invoke(core.clj:5895)
	at user$eval20$fn__29.invoke(form-init82057072006848720.clj:1)
	at user$eval20.invokeStatic(form-init82057072006848720.clj:1)
	at user$eval20.invoke(form-init82057072006848720.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:6927)
	at clojure.lang.Compiler.eval(Compiler.java:6917)
	at clojure.lang.Compiler.eval(Compiler.java:6917)
	at clojure.lang.Compiler.load(Compiler.java:7379)
	at clojure.lang.Compiler.loadFile(Compiler.java:7317)
	at clojure.main$load_script.invokeStatic(main.clj:275)
	at clojure.main$init_opt.invokeStatic(main.clj:277)
	at clojure.main$init_opt.invoke(main.clj:277)
	at clojure.main$initialize.invokeStatic(main.clj:308)
	at clojure.main$null_opt.invokeStatic(main.clj:342)
	at clojure.main$null_opt.invoke(main.clj:339)
	at clojure.main$main.invokeStatic(main.clj:421)
	at clojure.main$main.doInvoke(main.clj:384)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at clojure.lang.Var.invoke(Var.java:383)
	at clojure.lang.AFn.applyToHelper(AFn.java:156)
	at clojure.lang.Var.applyTo(Var.java:700)
	at clojure.main.main(main.java:37)
Caused by: java.lang.ClassNotFoundException: metabase.query_processor.interface.Field
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at clojure.lang.DynamicClassLoader.findClass(DynamicClassLoader.java:69)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at clojure.lang.DynamicClassLoader.loadClass(DynamicClassLoader.java:77)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at clojure.lang.RT.classForName(RT.java:2168)
	at clojure.lang.RT.classForNameNonLoading(RT.java:2181)
	at metabase.driver.generic_sql$loading__5569__auto____23043.invoke(generic_sql.clj:1)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.AFn.applyTo(AFn.java:144)
	at clojure.lang.Compiler$InvokeExpr.eval(Compiler.java:3652)
	... 63 more
Compilation failed: Subprocess failed
`

--- a/package.json
+++ b/package.json
@@ -139,6 +139,7 @@
     "sauce-connect-launcher": "^1.1.1",
     "selenium-webdriver": "^2.53.3",
     "style-loader": "^0.19.0",
+    "uglifyjs-webpack-plugin": "^1.0.0",
     "unused-files-webpack-plugin": "^3.0.0",
     "webchauffeur": "^1.2.0",
     "webpack": "^3.8.1",

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -11,6 +11,7 @@ var HtmlWebpackPlugin = require('html-webpack-plugin');
 var HtmlWebpackHarddiskPlugin = require('html-webpack-harddisk-plugin');
 var UnusedFilesWebpackPlugin = require("unused-files-webpack-plugin").default;
 var BannerWebpackPlugin = require('banner-webpack-plugin');
+var UglifyJSPlugin = require('uglifyjs-webpack-plugin')
 
 var fs = require('fs');
 
@@ -243,13 +244,15 @@ if (NODE_ENV !== "production") {
     config.output.devtoolModuleFilenameTemplate = '[absolute-resource-path]';
     config.output.pathinfo = true;
 } else {
-    config.plugins.push(new webpack.optimize.UglifyJsPlugin({
-        mangle: {
-            // this is required to ensure we don't minify Chevrotain token identifiers
-            // https://github.com/SAP/chevrotain/tree/master/examples/parser/minification
-            except: allTokens.map(function(currTok) {
-                return chevrotain.tokenName(currTok);
-            })
+    config.plugins.push(new UglifyJSPlugin({
+        uglifyOptions: {
+            mangle: {
+                // this is required to ensure we don't minify Chevrotain token identifiers
+                // https://github.com/SAP/chevrotain/tree/master/examples/parser/minification
+                except: allTokens.map(function(currTok) {
+                    return chevrotain.tokenName(currTok);
+                })
+            }
         }
     }))
 

